packages/cli/src/oclif/commands/promote.js

```diff
@@ -60,19 +61,24 @@ class PromoteCommand extends BaseCommand {
       `Preparing to promote version ${version} of your integration "${app.title}".`
     );
 
-    const body = {};
+    const body = {
+      job: {
+        name: 'promote',
+        to_version: version,
+      },
+    };
     if (changelog) {
-      body.changelog = changelog;
+      body.job.changelog = changelog;
```

Comment:
I know you were just following existing code, but you can move this above for the same effect:

```js
const body = {
      job: {
        name: 'promote',
        to_version: version,
        changelog,
      },
    };
```

Since it'll be undefined if it's missing, it just works